ovirt/data_source_ovirt_vms.go

```diff
@@ -183,7 +189,7 @@ func dataSourceOvirtVMsRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 	vms, ok := vmsResp.Vms()
-	if !ok || len(vms.Slice()) == 0 {
+	if !ok || (len(vms.Slice()) == 0 && allowEmptyEesult == false) {
```

Comment:
consider  using ! operator here ?

Reply:
Of course, I tried this at the beginning:
`if !ok || (len(vms.Slice()) == 0 && !allowEmptyEesult) {`

But GO compilator return error in this case:
```bash
# GOOS=linux GOARCH=amd64 go build
# github.com/ovirt/terraform-provider-ovirt/ovirt
ovirt/data_source_ovirt_vms.go:192:38: invalid operation: ! interface {}
```

Therefore I forced to use syntax: `allowEmptyEesult == false`